### DISCUSSION TITLE
Using the doc, I can't open a chest with a computer

### DISCUSSION THREAD

**Discussion by Alexis-Fayeulle** [cc-tweaked/CC-Tweaked#1661] [Q&A] [answered]
Using the doc
I can't open a chest with a computer

## How to reproduce :
1. Install minecraft 1.12.2 forge and CC: Tweaked with curseforge
2. Create world in creative
3. Get a chest and Advanced computer
4. Put chest and computer next to each other
5. Get on https://tweaked.cc/generic_peripheral/inventory.html
6. Scroll a little bit and get this code*
7. Paste the code in a program
8. Run

*The code
```
local chest = peripheral.find("minecraft:chest")
for slot, item in pairs(chest.list()) do
  print(("%d x %s in slot %d"):format(item.count, item.name, slot))
end
```

What i did wrong?
Why i can't open inventory like the doc?
Is my computer missing something?

<img width="1004" alt="Capture d’écran 2023-12-30 à 11 36 47" src="https://github.com/cc-tweaked/CC-Tweaked/assets/58552341/512f3292-0529-4a02-81d9-cd8111db1482">

**Comment by Wojbie** [comment, marked as answer]
Sadly 1.12.2 forge CC:T don't have inventory mechanics on its own. You needed to install Plethora mod to add that.

Note in doc it says it was added in version 1.94.0

**Reply by Alexis-Fayeulle** [reply]
Thanks =)
